oat-sa/tao-test-runner-qti-fe
# [TAO-9575] WTP-1413 - "time_remaining" in json is incorrect for exit_code 704

Related to : https://oat-sa.atlassian.net/browse/TAO-9575
 
Update item duration before sending it to server

#### How to test
 
- Prepare an instance of TAO with taoAct extension
- Prepare a delivery with multiple items and time limit
- Execute the delivery as a test taker
- Pass several question and wait until the end of time
- Make sure that in ODS Transmission Monitoring completedTestEvent has 0 time_remaining
